spring-cloud-kubernetes-client-autoconfig/src/test/java/org/springframework/cloud/kubernetes/client/KubernetesClientHealthIndicatorTests.java

```diff
@@ -43,48 +54,32 @@
 	private PodUtils<V1Pod> utils;
 
 	@Test
-	void getDetailsNotInsideTest() throws Exception {
+	void getDetailsNotInsideTest() {
 		when(utils.currentPod()).thenReturn(() -> null);
 		KubernetesClientHealthIndicator healthIndicator = new KubernetesClientHealthIndicator(utils);
-		assertThat(healthIndicator.getDetails().containsKey(KubernetesClientHealthIndicator.INSIDE)).isTrue();
-		assertThat(healthIndicator.getDetails().get(KubernetesClientHealthIndicator.INSIDE)).isEqualTo(false);
+		Map<String, Object> details = healthIndicator.getDetails();
+
+		assertThat(details.containsKey(INSIDE)).isTrue();
+		assertThat(details.get(INSIDE)).isEqualTo(false);
 	}
 
 	@Test
-	void getDetailsInsideTest() throws Exception {
-		Map<String, String> labels = new HashMap<>();
```

Comment:
I've moved the stubbing of the pod, as well as the field names it contained to `StubProvider`. This is used in both health and info tests.